contrib/_test.sh

```diff
@@ -23,7 +23,7 @@ fi
 # Test if panic in C code aborts the process (either with a real panic or with SIGILL)
 cargo test -- --ignored --exact 'tests::test_panic_raw_ctx_should_terminate_abnormally' 2>&1 \
     | tee /dev/stderr \
-    | grep "SIGILL\\|\[libsecp256k1] illegal argument. !rustsecp256k1_v0_._._fe_is_zero(&ge->x)"
+    | grep "SIGILL\\|\[libsecp256k1] illegal argument. "
```

Comment:
Why this change? It works still, right? You just don't like the grep statement?

Reply:
No, the assertion has changed in the latest version of libsecp.

Reply:
Oh, cool. Thanks.